README.md

```diff
@@ -52,6 +52,17 @@ maintainable, SIG-Addons will perform periodic reviews and deprecate
 contributions which will be slated for removal. More information will
 be available after we submit a formal request for comment.
 
+## Continuous build status
+
+### Official Builds
+
+| Build Type      | Status |
```

Comment:
IMO this can probably get moved up to right below the other badges

Reply:
Done.